src/utils/binder.js

```diff
@@ -0,0 +1,21 @@
+import { bindKey, rest } from 'lodash';
+
+export default class Binder {
+  constructor() {
+    this.disposal = [];
+  }
+
+  on(emitter, event, handler) {
```

Comment:
This implementation is technically a BC break since it doesn't spread any additional args - do you envision any issue with that?  I can't think of any reasonable use-case to pass additional args.